google/cloud/bigquery/v2/minimal/internal/job_logging.cc

```diff
@@ -32,12 +33,15 @@ template <typename Functor, typename Request,
               Functor, rest_internal::RestContext&, Request const&>>
 Result JobLogWrapper(Functor&& functor, rest_internal::RestContext& context,
                      Request const& request, char const* where,
+                     absl::string_view request_name,
+                     absl::string_view response_name,
                      TracingOptions const& options) {
   auto formatter = [](std::string* out, auto const& header) {
     absl::StrAppend(out, "name=", header.first, ", value={",
                     absl::StrJoin(header.second, "&"), "}");
   };
-  GCP_LOG(DEBUG) << where << "() " << request << ", Context={"
+  GCP_LOG(DEBUG) << where << "() << "
+                 << request.DebugString(request_name, options) << ", Context={"
                  << internal::DebugString(
```

Comment:
And to the end, you may like to think about how things are bracketed/quoted.

I would vote for something like ...
```
  Context { { name: "name1" value: "value1" } { name: "name2" value: "value2" } }
```
which would more closely align with the request/response logging.